.gitignore

```diff
@@ -9,3 +9,4 @@
 **/.classpath
 **/.idea
 **/*.iml
+out
```

Comment:
Is 'out' a folder generated by IntelliJ?

Reply:
I think it is useless here. An out directory is created for each sub project. I dont know why, it may the OSGi plugin:

![image](https://user-images.githubusercontent.com/6577027/132033167-ef7ed279-4c76-4339-922d-6b507ede1945.png)